tests/models_test.py

```diff
@@ -357,11 +621,18 @@ def test_change_const(self):
         assert t.CONST.change_const(False).is_var()
 
     def test_merge(self):
-        assert t.NODE_CONST.merge(t.NODE_CLASS) == t.NODE_CONST_CLASS
+        assert t.UNKNOWN.merge(t.NODE) == t.NODE
+        assert t.NODE.merge(t.CONST_NODE) == t.CONST_NODE
+        assert t.CONST_NODE.merge(t.CONST_NODE_CLASS) == t.CONST_NODE_CLASS
+        assert t.NODE.merge(t.NODE_LINK_CLASS) == t.NODE_LINK_CLASS
+        assert t.CONNECTOR.merge(t.CONST_MEMBERSHIP_ARC) == t.CONST_MEMBERSHIP_ARC
         with pytest.raises(InvalidTypeError, match=CommonErrorMessages.INVALID_TYPE.value):
```

Comment:
there is a chance that only first statement in this block is executed, can you split it into multiple `with pytest.raises` blocks?

Reply:
it was fixed. Resee it, please